### PR TITLE
6lowpan: make SLAAC work

### PR DESCRIPTION
**Pull request by OlegHahm**
I'm not quite sure about the initial purpose of the `GNRC_IPV6_NETIF_ADDR_FLAGS_TENTATIVE`, but I couldn't find a real use for it, while it causes trouble if a node tries to register a new address.

### CODE REVIEW

**Comment by authmillenon** [comment]
It is used to mark an address for SLAAC where the duplicity isn't confirmed or unconfirmed yet. See https://tools.ietf.org/html/rfc4862#page-13. If it however causes problems at registration I'm fine with it "breaking" the RFC for a few microseconds, until we find a more solid solution.

File: sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c [hunk 1]
```diff
@@ -106,6 +105,13 @@ static ipv6_addr_t *_add_addr_to_entry(gnrc_ipv6_netif_t *entry, const ipv6_addr
     }
     else {
         if (!ipv6_addr_is_link_local(addr)) {
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER
+            tmp_addr->valid = 0xFFFF;
+            gnrc_sixlowpan_nd_router_abr_t *abr = gnrc_sixlowpan_nd_router_abr_get();
+            if (gnrc_sixlowpan_nd_router_abr_add_prf(abr, entry, tmp_addr) < 0) {
+                DEBUG("ipv6_netif: error adding prefix to 6LoWPAN-ND management\n");
+            }
+#endif
```
Comment (authmillenon) [inline sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c]: Why did you move that?

**Comment by authmillenon** [comment]
Apart from the one comment (which is more a question than a remark): ACK.

File: sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c (hunk 1 above)
Comment (authmillenon) [inline sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c]: The answer lies hidden in the commit message ;)

**Comment by OlegHahm** [comment]
> It is used to mark an address for SLAAC where the duplicity isn't confirmed or unconfirmed yet. See https://tools.ietf.org/html/rfc4862#page-13. If it however causes problems at registration I'm fine with it "breaking" the RFC for a few microseconds, until we find a more solid solution. 

To be more concrete about the problem: a node adds a new address based on a received PIO and tries to register it via NS, but won't accept the corresponding NA since it is sent to a "tentative" address.

**Comment by authmillenon** [comment]
Yeah, I thought it might be something like that.

**Comment by OlegHahm** [comment]
btw with this PR, SLAAC seems to work like charm :smiley:

**Comment by authmillenon** [comment]
Then let's merge this as soon as Travis is happy :)
